src/utilities/viewer.cpp

```diff
@@ -990,7 +1003,28 @@ LRESULT CALLBACK TestViewerWindow::WindowProc(
     case WM_KEYUP:
       gInstance->OnKeyUp(wParam);
       return DefWindowProc(hWnd, uMsg, wParam, lParam);
+    case WM_SIZE:
+      [[fallthrough]];
+    case WM_MOVE: {

```

Comment:
braces are only needed for case statements when there's local variable definitinos

Reply:
got it, had those initially, and it complained about not having the braces, removed the variables in a "cleanup pass", not knowing how what or why the braces were required (scoping), good to know :)